### ISSUE TITLE
va-checkbox announces twice in JAWS browse mode

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#4538]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The Decision Reviews team received the following feedback in a 508 audit of the 0995 Supplemental Claim form, that uses the VA Checkbox. 

> Throughout the product, the names for check boxes are read twice when reading with the arrow keys. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), “I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground. check box not checked I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground.” is read. (New: 04/04/2025)

## What I expected to happen
Should be able to navigate to each checkbox with label once via arrow key navigation with JAWS

## Reproducing

- CSS Library version:
- Device: (e.g. iPhone 8, Macbook) JAWS
- Browser: (e.g. Firefox, IE 11) Edge

Steps to reproduce:

1. Can be replicated on the storybook link for the component https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox-group--docs
2. Turn on JAWS and navigate to the checkboxes in browse mode. Use the arrow keys to navigate through the checkboxes
3. You will be able to land on the input and the label individually with separate announcements

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

### Details
This issue was identified in a 508 audit - see third item https://github.com/department-of-veterans-affairs/va.gov-team/issues/107044

**Comment by coforma-jamie** [comment]
Another 508 audit called this out (Defect 16): https://github.com/department-of-veterans-affairs/va.gov-team/issues/131625
